crates/napi/src/bindgen_runtime/js_values/bigint.rs

```diff
@@ -226,20 +240,19 @@ impl From<u64> for BigInt {
 impl From<i128> for BigInt {
   fn from(val: i128) -> Self {
     let sign_bit = val < 0;
-    let words = (if sign_bit { -val } else { val }).to_ne_bytes();
+    let val = if sign_bit { -val } else { val };
     BigInt {
       sign_bit,
-      words: unsafe { std::slice::from_raw_parts(words.as_ptr() as *mut _, 2).to_vec() },
+      words: vec![val as _, (val >> 64) as _],
```

Comment:
This change does not change the asm output as well.
[before](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=40347958aeba56d3230ff74296f47898), [after](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=faa7513e082d68d2fc005951893ff09a)